.devcontainer/devcontainer.json

```diff
@@ -25,7 +24,9 @@
       // Please keep this file in sync with settings in home-assistant/.vscode/settings.default.json
       "settings": {
         "python.experiments.optOutFrom": ["pythonTestAdapter"],
-        "python.pythonPath": "/usr/local/bin/python",
+        "python.defaultInterpreterPath": "/home/vscode/.local/ha-venv/bin/python",
+        "python.pythonPath": "/home/vscode/.local/ha-venv/bin/python",
+        "python.terminal.activateEnvInCurrentTerminal": true,
```

Comment:
This setting would maybe a good one for the settings.default.json as well

Reply:
We should add it in a follow up PR. This PR is already big enough